.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on: [push, pull_request] # on all pushes and PRs
+
+jobs:
+  ros1_bridge:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        distro: ["ros:rolling-ros1-bridge-focal", "ros:foxy-ros1-bridge-focal"]
```

Comment:
The `master` branch is not targeting Foxy, so I would only target Rolling for CI here. Later, we can port this PR to the [foxy](https://github.com/ros2/ros1_bridge/tree/foxy) branch, making the necessary changes to target Foxy.

Reply:
now targets only `rolling`